packages/import/src/index.ts

```diff
@@ -360,9 +198,246 @@ function visitFile(
       }
     }
   }
-  return visitedFiles.get(filePath);
+  return visitedFiles.get(filePath)!;
+}
+
+export function extractDependencies(
+  filePath: string,
+  fileContents: string
+): {
+  definitionsByName: Map<string, Set<DefinitionNode>>;
+  dependenciesByDefinitionName: Map<string, Set<string>>;
+} {
+  const definitionsByName = new Map<string, Set<DefinitionNode>>();
+  const dependenciesByDefinitionName = new Map<string, Set<string>>();
+
+  const { document } = parseGraphQLSDL(filePath, fileContents, {
```

Comment:
I switched to using `parseGraphQLSDL` instead of `parse` because the former handles empty documents. This allows us to simplify the logic a bit.

Previously we had this logic to handle empty documents:

https://github.com/ardatan/graphql-tools/blob/34f1efd9c1ae45e3532f860d9a9ec6e1159be49e/packages/import/src/index.ts#L116-L122

which was actually buggy and would fail for a file like this:

```
# import "some-file"
# some comment
```

which is technically an empty document, but `otherLines` is `"# some comment"` so it's not an empty string.